htdocs/includes/db_mysql_lib.php

```diff
@@ -68,14 +68,13 @@ function query_alter($query)
         }
 }
 
-function query_associative_all( $query, $row_count ) 
+function query_associative_all($query) 
 {
     global $con;
 	if( !($result = mysql_query( $query, $con ) ) ) 
 	{
         return null;
     }
-	$row_count = mysql_num_rows( $result );
     $retval = array();

```

Comment:
This is the root of the change! As you can see, `$row_count` is overwritten, and then the result of the query is never used.

In fact, eliminating the `mysql_num_rows` call here might actually lead to some speed gains.